esp32/mptask.c

```diff
@@ -410,8 +410,7 @@ STATIC void mptask_update_lpwan_mac_address (void) {
 #endif
 
 STATIC void mptask_enable_wifi_ap (void) {
-    wlan_setup (WIFI_MODE_AP, DEFAULT_AP_SSID, WIFI_AUTH_WPA2_PSK, DEFAULT_AP_PASSWORD,
-                DEFAULT_AP_CHANNEL, ANTENNA_TYPE_INTERNAL, true);
+    wlan_setup (WIFI_MODE_AP, DEFAULT_AP_SSID, WIFI_AUTH_WPA2_PSK, DEFAULT_AP_PASSWORD, DEFAULT_AP_CHANNEL, ANTENNA_TYPE_INTERNAL, true, "illysky");
```

Comment:
the `illysky` default should be changed to something more appropriate to all users

Reply:
Ah yes! Sorry forgot about that. Perhaps “Pycom” or “MicroPython”.